integrations/postgresql/postgresql-health-check/sensu-integration.yaml

```diff
@@ -0,0 +1,115 @@
+---
+api_version: catalog/v1
+type: Integration
+metadata:
+  namespace: postgresql
+  name: postgresql-health-check
+spec:
+  class: supported # community, supported, enterprise, or partner
+  provider: monitoring
+  display_name: "PostgreSQL health check"
+  short_description: "Health check for the Sensu Enterprise datastore"
+  supported_platforms:
+    - darwin
+    - linux
+    - windows
+  tags:
+    - postgresql
+    - pgsql
+    - postgres
+    - database
+    - sensu
+  contributors:
+    - "@sensu"
+    - "@asachs01"
+  prompts:
+    - type: question
+      name: url
+      required: false
+      input:
+        type: string
+        title: Sensu Backend URL
+        description: >-
+          Please input the full path to your Sensu backend's `/health` endpoint (example: http://127.0.0.1:8080/health)
+        format: url
+        default: http://127.0.0.1:8080/health
+    - type: question
+      name: interval
+      required: false
+      input:
+        type: integer
+        title: Interval
+        description: >-
+          How often (in seconds) do you want to check the Sensu Enterprise datastore's health?
+        default: 30
+    - type: section
+      title: Pipeline Configuration
+    - type: markdown
+      body: |
+        Configure one or more [pipelines] for processing {{integration_short_name}} data.
+
+        [pipelines]: https://docs.sensu.io/sensu-go/latest/observability-pipeline/
+    - type: question
+      name: alert_pipeline
+      required: false
+      input:
+        type: string
+        title: Alert Pipeline
+        description: >-
+          How do you want to be alerted for failures detected by this pipeline (e.g. Slack or Microsoft Teams)?
+        ref: core/v2/pipeline/metadata/name
+        refFilter: .labels.provider == "alerts"
+    - type: question
+      name: incident_pipeline
+      required: false
+      input:
+        type: string
+        title: Incident Management Pipeline
+        description: >-
+          How do you want to process incidents for failures detected by this pipeline (e.g. Atlassian JIRA/ServiceDesk, or Pagerduty)?
+        ref: core/v2/pipeline/metadata/name
+        refFilter: .labels.provider == "incidents"
+    - type: question
+      name: metrics_pipeline
+      required: false
+      input:
+        type: string
+        title: Metrics Pipeline
+        description: >-
+          How do you want to process metrics collected by this integration?
+        ref: core/v2/pipeline/metadata/name
+        refFilter: .labels.provider == "metrics"
+  resource_patches:
+    - resource:
+        api_version: core/v2
+        type: CheckConfig
+        name: postgresql-health-check
+      patches:
+          - path: /metadata/name
+            op: replace
+            value: postgresql-healthcheck-[[auto_suffix]]
```

Comment:
What is this auto_suffix? It's not defined as a prompted variable so refering to it is a noop

Also do you really want to change the check name like this? Not seeing why this is needed

Reply:
Related: 

* We implemented `unique_id` (an 8-digit hex value) in https://github.com/sensu/sensu-enterprise-go/pull/2109 
* This closed the original `unique_id` proposal in https://github.com/sensu/sensu-enterprise-go/issues/2090 
* Somewhere along the way I documented this as `auto_suffix` (I think this was the original proposal, which eventually got changed to `unique_id`)